src/components/TabbedCard/TabbedCard.react.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import * as React from "react";
+import Tab from "./Tab.react";
+import Card from "../Card/Card.react";
+
+type Props = {|
+  +initialTab: string,
+  +children: React.ChildrenArray<React.Element<typeof Tab>>,
+|};
+
+type State = {|
+  selectedTitle: string,
+|};
+
+class TabbedCard extends React.PureComponent<Props, State> {
+  state = {
+    selectedTitle: this.props.initialTab,
+  };
+
+  render(): React.Node {
+    const tabs = React.Children.toArray(this.props.children);
+    return (
+      <Card>
+        <Card.Header>
+          <ul className="nav nav-tabs TabbedCard_header_tabs">
+            {tabs.map(tab => {

```

Comment:
Is it worth moving the logic and returning of the component inside here to the new Tab component? Then being able to do something like

```jsx
{tabs.map(tab => <Tab 
  title={title}
  active={title === this.state.selectedTitle} 
  onClick={() => this.setState({ selectedTitle: title })} 
/>)}
```

Of if we want to provide a bit more flexibility:
```jsx
{tabs.map(tab => React.cloneElement(tab, {
  active: { title === this.state.selectedTitle } 
  onClick: { () => this.setState({ selectedTitle: title }) } 
 })
)}
```